### PR TITLE
Point the telemetry service to telemetry.trento.suse.com

### PR DESCRIPTION
**Pull request by dottorblaster**
As above :sweat_smile:

### CODE REVIEW

File: web/telemetry/publisher.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ func (tp *TelemetryPublisher) Publish(telemetryName string, installationID uuid.
 	return nil
 }
 
-var apiHost = "https://telemetry.trento-project.io" // this service still does not exist
+var apiHost = "https://telemetry.trento.suse.com" // this service still does not exist
```
Comment (arbulu89) [inline web/telemetry/publisher.go:43]: Hey, @dottorblaster ,
I would remove the comment besides the new domain, as now it will exists hehe

**Review by arbulu89 — APPROVED** [review]

File: web/telemetry/publisher.go (hunk 1 above)
Comment (nelsonkopliku) [inline web/telemetry/publisher.go:43]: We can remove the comment I guess, that service is being spawned up as I understood.

**Review by nelsonkopliku — APPROVED** [review]

File: web/telemetry/publisher.go (hunk 1 above)
Comment (arbulu89) [reply web/telemetry/publisher.go:43]: By the way, now I notice, `apiHost` looks a quite inaccurate name for this, doesn't it? XD

Comment (dottorblaster) [reply web/telemetry/publisher.go:43]: Fair 'nuff

Comment (nelsonkopliku) [reply web/telemetry/publisher.go]: Bad naming here is my fault :sweat_smile: 
Feel free to change or not to something like `telemetryApiHost` `telemetryEndpoint` or whatever makes sense.

Comment (dottorblaster) [reply web/telemetry/publisher.go]: Comment removed, thanks for that! `apiHost` was quite clear to me but feel free to suggest something else, I think this line is going to be edited a lot to allow configuration via build flags in the future so... :smile:

Comment (stefanotorresi) [reply web/telemetry/publisher.go]: 👍🏼 for renaming. I suggest `telemetryServiceUrl`.

**Pull request merged**